projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -975,6 +975,11 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         }
     }
 
+    public onInputGroupClick(event: MouseEvent, target?: HTMLElement) {
```

Comment:
This can be `event.currentTarget` so you don't need two params

Reply:
we actually do not need this anymore